### PR TITLE
reveiw_create _contact_with person

### PR DESCRIPTION
**Pull request by ghada-37**

### CODE REVIEW

File: ui_testing/testcases/basic_tests/test05_contacts.py [hunk 1]
```diff
@@ -142,25 +142,30 @@ def test_006_download_contact_sheet(self):
...
     def test_007_create_contact_with_person(self):
+        """
+        New: Contact: Creation Approach: I can create new contact successfully with contact person
+        LIMS-6386
+        """
         contact_data = self.contact_page.create_update_contact()
-
-        self.base_selenium.LOGGER.info('filter by contact no.: {} to get the record'.format(contact_data['Contact No']))
+        self.contacts_page.info('filter by contact no.: {} to get the record'.format(contact_data['Contact No']))
         self.base_selenium.refresh()
```
[the remark this answers is not in the document]
Comment (ghada-37) [reply ui_testing/testcases/basic_tests/test05_contacts.py]: when I remove it I got :
======================================================================
ERROR: New: Contact: Creation Approach: I can create new contact successfully with contact person
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ghada/1lims-automation/ui_testing/testcases/basic_tests/test05_contacts.py", line 153, in test_007_create_contact_with_person
    self.contact_page.open_edit_page(row=contact_record)
  File "/home/ghada/1lims-automation/ui_testing/pages/base_pages.py", line 148, in open_edit_page
    row.find_element_by_xpath(xpath).click()
  File "/home/ghada/.local/lib/python3.6/site-packages/selenium/webdriver/remote/webelement.py", line 351, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ghada/.local/lib/python3.6/site-packages/selenium/webdriver/remote/webelement.py", line 659, in find_element
    {"using": by, "value": value})['value']
  File "/home/ghada/.local/lib/python3.6/site-packages/selenium/webdriver/remote/webelement.py", line 633, in _execute
    return self._parent.execute(command, params)
  File "/home/ghada/.local/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ghada/.local/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//span[@class="mr-auto"]/a"}
  (Session info: chrome=79.0.3945.88)


----------------------------------------------------------------------
Ran 1 test in 158.459s

FAILED (errors=1)

[the remark this answers is not in the document]
Comment (ghada-37) [reply ui_testing/testcases/basic_tests/test05_contacts.py]: I don't know why option button not seen before refreshing !

File: ui_testing/testcases/basic_tests/test05_contacts.py [hunk 2]
```diff
@@ -140,28 +140,31 @@ def test_006_download_contact_sheet(self):
...
         contact_data = self.contact_page.create_update_contact()
-
-        self.base_selenium.LOGGER.info('filter by contact no.: {} to get the record'.format(contact_data['Contact No']))
+        self.info('filter by contact no.: {} to get the record'.format(contact_data['Contact No']))
         self.base_selenium.refresh()
 
         contact_record = self.contact_page.search(value=contact_data['Contact No'])[0]
-        self.base_selenium.LOGGER.info('open the record in edit to compare the data')
+        self.contacts_page.info('open the record in edit to compare the data')
         self.contact_page.open_edit_page(row=contact_record)
```
Comment (SalmaaRagab) [inline ui_testing/testcases/basic_tests/test05_contacts.py]: @ghada-37 Try this and let me know
```suggestion
        self.contact_page.open_edit_page_by_css_selector(row=contact_record)
```

File: ui_testing/testcases/basic_tests/test05_contacts.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (SalmaaRagab) [reply ui_testing/testcases/basic_tests/test05_contacts.py]: @ghada-37  Check out the suggestion I've made on line 153 [https://github.com/modeso-open-source/1lims-automation/pull/165#discussion_r368977553](url)

[the remark this answers is not in the document]
Comment (ghada-37) [reply ui_testing/testcases/basic_tests/test05_contacts.py]: thank alot @SalmaaRagab  
it worked finally

**Pull request merged**